content/blog/2019/08/2019-08-17-folder-auth-plugin.adoc

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title: "Introducing Folder Auth Plugin v1.0"
+tags:
+- plugins
+- security
+- performance
+- gsoc
+- gsoc2019
+author: abhyudayasharma
+---
+
+During my link:/projects/gsoc/2019/role-strategy-performance[Google Summer of Code Project],
+I have created the brand new plugin:folder-auth[Folder Auth Plugin] for easily
+managing permissions to projects organized in folders from the plugin:cloudbees-folder[Folders plugin].
```

Comment:
It would be great to have a kind of problem statement in the beginning. Why was it needed?